molecule/internal-tls/molecule.yml

```diff
@@ -0,0 +1,40 @@
+---
+driver:
+  name: ${CONTAINER_BIN:-docker}
+platforms:
+  - name: qdrtest
+    image: docker.io/tripleomaster/centos-binary-qdrouterd:current-tripleo-rdo
```

Comment:
To avoid odd failures due to pull limit requests, might want to look at using quay.io. It looks like there is a sync there now you could make use of.

https://quay.io/repository/tripleomaster/openstack-qdrouterd?tab=tags

Reply:
Oh thanks, I'll look at changing that over.

Reply:
This seems to work with a simple switch to `quay.io/tripleomaster/openstack-qdrouterd:current-tripleo` across the whole repo. I haven't researched the precise difference in lineage between these two images but the naming suggests that this one from quay is further upstream (pre-RDO) so perhaps it's a better match anyways regardless of pull limits?